docs/source/examples.rst

```diff
@@ -0,0 +1,267 @@
+Class Example Usage
+===================
+
+``waterdynamics`` operates around class based analyses.
+Find examples of their use below.
+
```

Comment:
include the SeeAlso about h-bonds from https://docs.mdanalysis.org/stable/documentation_pages/analysis/waterdynamics.html#hydrogenbondlifetimes